BasicTest_dnxcore50/project.json

```diff
@@ -1,32 +1,31 @@
 {
   "version": "1.0.0-*",
   "description": "StackExchange.Redis.BasicTest dnxcore50",
-  "authors": [ "" ],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
-
-  "compile": [
-    "../BasicTest/Program.cs"
-  ],
+  "buildOptions": {
+    "compile": {
+      "include": [
+        "../BasicTest/Program.cs"
+      ]
+    }
+  },
   "dependencies": {
     "StackExchange.Redis": {
-      "version": "1.1.572-alpha",
+      "version": "1.1.*",
       "target": "project"
     }
   },
   "commands": {
     "run": "BasicTest_dnxcore50"
   },
-
-  
   "frameworks": {
-    "dnxcore50": {
+    "netcoreapp1.0": {
+      "buildOptions": {
+        "define": [ "CORE_CLR" ]
```

Comment:
The compiler defines a variable named `NETCOREAPP1_0` in case you were looking for it.